package exception.user;

import exception.common.ConflictException;

public class UserAlreadyExistsException extends ConflictException {
}
